reviewrot/basereview.py

```diff
@@ -82,6 +87,64 @@ def check_request_state(self, created_at,
                 raise ValueError("Invalid duration type: %s" % duration)
         return True
 
+    def _decode_response(self, response):
+        """
+        Remove Gerrit's prefix and convert to JSON.
+        Returns:
+            Converted JSON content
+        Raises:
+            ValueError if the content is not in proper json format.
+        """
+        gerrit_json_prefix = ")]}\'\n"
+        content = response.content.strip()
+        try:
+            if response.encoding:
+                content = content.decode(response.encoding)
+            response.raise_for_status()
+            content_type = response.headers.get('content-type', '')
+            if content_type.split(';')[0] != 'application/json':
```

Comment:
Since I am adding Content-Type in header, I believe there is no need to do this check. Correct me if I am wrong.